packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ProxyEOA.sol

```diff
@@ -22,48 +20,34 @@ contract OVM_ProxyEOA {
      * Constants *
      *************/
 
+    address constant DEFAULT_IMPLEMENTATION = 0x4200000000000000000000000000000000000003;
     bytes32 constant IMPLEMENTATION_KEY = 0xdeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddead;
```

Comment:
Maybe for a separate PR, but harping again that this should really use [this EIP](https://eips.ethereum.org/EIPS/eip-1967) here.

Reply:
I agree that we should use that EIP, note that this will also break our regenesis script
https://github.com/ethereum-optimism/scripts/blob/main/scripts/state-surgery.js
Will add an issue over there

Reply:
https://github.com/ethereum-optimism/scripts/issues/12

Reply:
Created an issue here: https://github.com/ethereum-optimism/optimism/issues/550